gitgud/skills/rampup/_detaching/solution.txt

```diff
@@ -0,0 +1,4 @@
+# User would instead check out the commit hash using only one command
+git checkout bugFix
+git checkout @^
+git checkout @{1}
```

Comment:
The solution we're looking for is something like `git checkout asdf`. For this level, we should override the default `Level.solution()` method  and output the correct commit hash.

The test case should call a function to get the correct solution.

Reply:
Hm...well, this PR is already pretty long, so I think it's worth making a different issue/PR to autogenerate the solutions.txt file for that solution. I'm leaving this as is.

Reply:
As long as we pass test cases, that's fine. I made a new issue in #215